boopickle/src/test/scala/org/http4s/booPickle/BoopickleSpec.scala

```diff
@@ -54,27 +53,23 @@ class BoopickleSpec extends Http4sSpec with BooPickleInstances {
 
   implicit val fruitEq: Eq[Fruit] = Eq.fromUniversalEquals
 
-  "boopickle encoder" should {
-    "have octet-stream content type" in {
-      encoder.headers.get(`Content-Type`) must_== Some(
-        `Content-Type`(MediaType.application.`octet-stream`))
-    }
+  test("have octet-stream content type") {
+    assertEquals(
+      encoder.headers.get(`Content-Type`),
+      Some(`Content-Type`(MediaType.application.`octet-stream`)))
   }
 
-  "booEncoderOf" should {
-    "have octect-stream content type" in {
-      booEncoderOf[IO, Fruit].headers.get(`Content-Type`) must_== Some(
-        `Content-Type`(MediaType.application.`octet-stream`))
-    }
+  test("have octect-stream content type") {
+    assertEquals(
+      booEncoderOf[IO, Fruit].headers.get(`Content-Type`),
+      Some(`Content-Type`(MediaType.application.`octet-stream`)))
   }
 
-  "booOf" should {
-    "decode a class from a boopickle decoder" in {
-      val result = booOf[IO, Fruit]
-        .decode(Request[IO]().withEntity(Banana(10.0): Fruit), strict = true)
-      result.value.unsafeRunSync() must_== Right(Banana(10.0))
-    }
+  test("decode a class from a boopickle decoder") {
+    val result = booOf[IO, Fruit]
+      .decode(Request[IO]().withEntity(Banana(10.0): Fruit), strict = true)
+    result.value.map(assertEquals(_, Right(Banana(10.0))))
   }
 
-  checkAll("EntityCodec[IO, Fruit]", EntityCodecTests[IO, Fruit].entityCodec)
+  // checkAll("EntityCodec[IO, Fruit]", EntityCodecTests[IO, Fruit].entityCodec)
```

Comment:
`EntityCodecTests` needs a rewrite to run on munit-cats-effect, if you think munit is fine I could try to do the rewrite

Reply:
Why does it need a rewrite?  I think it's based just on Discipline?

I'm for what makes it work.  I just haven't studied it enough to understand.

Reply:
The particular laws take an `Eq[IO[A]]` which we don't have right now